src/metrics/metrics.go

```diff
@@ -81,7 +81,7 @@ func CollectBrokerRequestMetrics(sample *metric.Set, metricSets []*JMXMetricSet,
 		// If we fail we don't want a total failure as other metrics can be collected even if a single failure/timout occurs
 		if err != nil {
 			if jmxErr, ok := gojmx.IsJMXError(err); ok {
-				log.Error("Unable to execute JMX query for MBean '%s': %v", beanName, jmxErr)
+				log.Warn("Unable to execute JMX query for MBean '%s': %v", beanName, jmxErr)
```

Comment:
🤔 I wonder if we should go further, a [similar error in nri-cassandra](https://github.com/newrelic/nri-cassandra/blob/master/src/metrics.go#L45) is logged with debug level. @paologallinaharbur WDYT?